vendor-extra/JatsContentBundle/src/EventListener/BuildView/FrontArticleTypeContentMetaListener.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener\BuildView;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\ContextAwareTranslation;
+use Libero\ViewsBundle\Views\SimplifiedViewConverterListener;
+use Libero\ViewsBundle\Views\View;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use function sprintf;
+
+final class FrontArticleTypeContentMetaListener
+{
+    use ContextAwareTranslation;
+    use SimplifiedViewConverterListener;
+
+    private $translationKeys;
+
+    public function __construct(TranslatorInterface $translator, array $translationKeys = [])
+    {
+        $this->translator = $translator;
+        $this->translationKeys = $translationKeys;
+    }
+
+    protected function handle(Element $object, View $view) : View
+    {
+        $items = $view->getArgument('items') ?? [];
+
+        if (isset($items['type'])) {
```

Comment:
is this the main check that avoids clashes between different listeners? This handles `<front>` as other listeners will probably do, but it will be the only one producing `type` (for this template?)

Reply:
Yep. It's not ideal, but means you don't have to have one listener add all the values (we know we'll have support for content type, publication date and issue name, which will take different forms). This makes it easier to override one bit.